package.json

```diff
@@ -27,9 +27,12 @@
     "@auth0/auth0-react": "^2.2.1",
     "@docusaurus/core": "^2.4.1",
     "@docusaurus/preset-classic": "^2.4.1",
+    "@mdx-js/react": "^1.6.21",
```

Comment:
I removed this dependency a long time ago, because it seemed to be unused. I had to add it back in so that the plugin would work....and I don't really know why, other than it helped resolved some react peer-dependency issues. 🤷